lib/t_web/channels/feed_channel.ex

```diff
@@ -234,7 +230,7 @@ defmodule TWeb.FeedChannel do
            audio_only: [_our, mate_audio_only],
            event: %{timestamp: timestamp}
          }} ->
-          expiration_date = timestamp |> DateTime.add(@match_ttl)
+          expiration_date = DateTime.add(timestamp, Matches.match_ttl())
```

Comment:
- [ ] `pre_voicemail_ttl`

Reply:
- [ ] `inserted_at`, `exchanged_voicemail`

Reply:
@akhrail1996 I've added more info in case of match:

```elixir
def handle_in("like", %{"user_id" => liked}, socket) do
    %{current_user: %{id: liker}} = socket.assigns

    # TODO check that we had a call?

    reply =
      case Matches.like_user(liker, liked) do
        {:ok, %{match: _no_match = nil}} ->
          :ok

        {:ok,
         %{
           match: %{id: match_id, inserted_at: inserted_at},
           audio_only: [_our, mate_audio_only]
         }} ->
          # TODO return these timestamps from like_user
          inserted_at = DateTime.from_naive!(inserted_at, "Etc/UTC")
          expiration_date = DateTime.add(inserted_at, Matches.pre_voicemail_ttl())

          {:ok,
           %{
             "match_id" => match_id,
             "audio_only" => mate_audio_only,
             "expiration_date" => expiration_date,
             "inserted_at" => inserted_at,
             "exchanged_voicemail" => false
           }}

        {:error, _step, _reason, _changes} ->
          :ok
      end

    {:reply, reply, socket}
  end
```